README.md

```
@@ -104,4 +104,48 @@
 5. To fix lint issues
     ```
     ruff check --fix
-    ```
\ No newline at end of file
+    ```
+
+## Debug Mode with VS Code
+
+### Prerequisites
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)